ipfs-cluster-service/main.go

```diff
@@ -194,15 +189,35 @@ func main() {
 		{
 			Name:  "init",
 			Usage: "create a default configuration and exit",
+			Description: fmt.Sprintf(`
+This command will initialize a new service.json configuration file
+for %s.
+
+By default, %s requires a cluster secret. This secret will be
+automatically generated, but can be manually provided with -custom-secret
+(in which case it will be prompted), or by setting the CLUSTER_SECRET
+environment variable.
+
+The private key for the libp2p node is randomly generated in all cases.
+
+Note that the -force first-level-flag allows to overwrite an existing
+configuration.
+`, programName, programName),
+			ArgsUsage: " ",
 			Flags: []cli.Flag{
 				cli.BoolFlag{
-					Name:  "gen-secret, g",
-					Usage: "automatically generate cluster secret",
+					Name:  "custom-secret, s",
```

Comment:
Commit message says that the short flag here is `m`, change either this or commit message to be consistent.